templates/d1-openapi/vitest.config.ts

```diff
@@ -0,0 +1,33 @@
+import path from "node:path";
+import { defineWorkersConfig, readD1Migrations } from "@cloudflare/vitest-pool-workers/config";
+
+export default defineWorkersConfig(async () => {
+  const migrationsPath = path.join(__dirname, 'drizzle/migrations');
```

Comment:
aside: does `__dirname` get shimmed automatically when we run vitest? (could cause problems with modern versions of node)

that also begs the question: what version of `node` are you running locally?

Reply:
ah apparently i've been using the terms "shim" and "polyfill" interchangeably but they're subtly different. i was referring to polyfilling __dirname for modern node versions. if you didn't hit issues on v22 it should be fine